src/client/common/application/webviewViews/webviewView.ts

```diff
@@ -38,5 +49,11 @@ export class WebviewView extends Webview implements IWebviewView {
                 this.panelOptions.listener.onMessage(message.type, message.payload);
             })
         );
+
+        this.disposableRegistry.push(
+            webviewHost.onDidChangeVisibility(() => {
```

Comment:
Does this fire when the host is launched? Seems like it should if it doesn't.

Reply:
@rchiodo Yeah, I think you are right. I don't believe that I noticed as my variable view does an initial direct check for visible state before it hooks up and watches for the event. But we should also fire an initial fire of this.